pkgs/development/python-modules/pydrive2/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, python3 }:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "pydrive2";
+  version = "1.6.0";
+
+  src = python3.pkgs.fetchPypi {
+    pname = "PyDrive2";
+    inherit version;
+    sha256 = "0b803dq6g13zypqplwi4bzdjmsdrxmvfiba9121wma51j83350kp";
+  };
+
+  doCheck = false; # the tests require your GCP credentials
+
+  propagatedBuildInputs = with python3.pkgs; [
+    google_api_python_client
```

Comment:
```suggestion
    google-api-python-client
```

Reply:
fixed in 5b86966d9a6cbb4d35419cd370d467727887bc40